### PR TITLE
Ship plugins system-wide

### PR DESCRIPTION
**Pull request by mathiascode**
Would be nice to have the plugins included by default, and potentially detect issues in them.

### CODE REVIEW

File: plugins/README [hunk 1]
```diff
@@ -1,20 +1,14 @@
...
-
-
 FOR END USERS
 
-In order to get the plugins working you need to copy them to
-~/.nicotine/plugins, creating this directory if it doesn't exist yet. Plugins
-will be loaded upon startup, and can be reloaded with /reload. This reload
-command does not always completely reload the plugins due to the way Python
-works, in some cases a restart of N+ is needed to get new versions working.
+In order to load your own plugins you need to copy them to
```
Comment (kiplingw) [inline plugins/README:6]: This actually should really be relevant more for developers than end users. End users should never have to do this. The plugins should ship within the deb and be correctly installed into the FHS.

Comment (kiplingw) [reply plugins/README:6]: When I think of vanilla "end users" I don't normally think of people writing their own plugins.

Comment (mathiascode) [reply plugins/README:6]: Good point.

**Comment by kiplingw** [comment]
@mathiascode, I'm not sure if you received the Launchpad builder's [build log failure](https://launchpadlibrarian.net/480829943/buildlog.txt.gz), but it appears this commit broke it.

**Pull request merged**